Caltech-IPAC/firefly
# Firefly-973, Firefly-974: Fixed Multiple Issues

#### [Firefly-973](https://jira.ipac.caltech.edu/browse/FIREFLY-973), [Firefly-974](https://jira.ipac.caltech.edu/browse/FIREFLY-974): Fixed Multiple issues
   - [Firefly-973](https://jira.ipac.caltech.edu/browse/FIREFLY-973) Fixed finder chart performance problem
       - updated the revisions of package.json libraries
       - optimized getLayouInfo() will not create a new object unless necessary
       - workers: when worker deleted it  will now fire a reject for all outstanding promises
       - better expired plot checking in RawDataOps
  - [Firefly-974](https://jira.ipac.caltech.edu/browse/FIREFLY-974): popout mouse readout not showing flux
  - _Bug fix_: FITS Information dialog was not showing file size
  - _Bug Fix_: Multi product viewer "All Images" option not showing the correct file when file changed by DataProductAnalyzer
  - _Bug Fix_: fixed non-passive event error warning coming from chrome
#### Testing
   - _finderchart:_ https://firefly-973-fc-performance.irsakudev.ipac.caltech.edu/applications/finderchart/
       - Load table of target, performance should be normal
       - Click on the info button for an image. You should see the file size.
   - _Sofia_: https://firefly-973-fc-performance.irsakudev.ipac.caltech.edu/applications/sofia
        - Flux information should show up on the popup mouse readout
        - It should not matter if you get at the first HDU from the pull down or form show "All Images"
   - _version_: 2022.1-DEV:HEAD_0497
